synapse/app/homeserver.py

```diff
@@ -488,6 +488,29 @@ def phone_stats_home(hs, stats, stats_process=_stats_process):
     if uptime < 0:
         uptime = 0
 
+    #
+    # Performance statistics
+    #
```

Comment:
I feel like this should have some comment saying to keep this at the top or something...?

Reply:
```suggestion
    #
    # Performance statistics. Keep this early in the function to maintain reliability of `test_performance_100` test.
    #
```
maybe?

Reply:
It's better than what I came up with...